src/style.css

```diff
@@ -1,3 +1,75 @@
+:root {
+  --link-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.nav-link {
+  display: flex;
+  list-style: none;
+  margin: 0 10px;
+  padding: 0;
+}
+
+.nav-link:first-child {
+  margin-left: 0;
+}
+
+.nav-link:last-child {
+  margin-right: 0;
```

Comment:
you set left and right margins for implementing space between links
you can set it with just one rule:
```
.nav-link:not(:last-child) {
  margin-right: 20px;
}
```